src/RaisedButton/RaisedButton.js

```diff
@@ -111,69 +111,68 @@ class RaisedButton extends React.Component {
 
   static propTypes = {
     /**
-     * Override the background color. Always takes precedence unless the button is disabled.
+     * This value will override the default background color for the button,
+     * but not the default disabled background color
+     * (use `disabledBackgroundColor` for this).
      */
     backgroundColor: React.PropTypes.string,
 
     /**
-     * This is what will be displayed inside the button.
-     * If a label is specified, the text within the label prop will
-     * be displayed. Otherwise, the component will expect children
-     * which will then be displayed. (In our example,
-     * we are nesting an `<input type="file" />` and a `span`
-     * that acts as our label to be displayed.) This only
-     * applies to flat and raised buttons.
+     * The content of the button.
+     * If a label is provided via the `label` prop, the text within the label
+     * will be displayed in addition to the content provided here.
      */
     children: React.PropTypes.node,
 
     /**
-     * The css class name of the root element.
+     * The CSS class name of the root element.
      */
     className: React.PropTypes.string,
 
     /**
-     * Disables the button if set to true.
+     * If true, the button will be disabled.
      */
     disabled: React.PropTypes.bool,
 
     /**
-     * Override the background color if the button is disabled.
+     * This value will override the default background color for the button
+     * when it is disabled.
      */
     disabledBackgroundColor: React.PropTypes.string,
 
     /**
-     * Color of the label if disabled is true.
+     * The color of the button's label when the button is disabled.
      */
     disabledLabelColor: React.PropTypes.string,
 
     /**
-     * If true, then the button will take up the full
-     * width of its container.
+     * If true, the button will take up the full width of its container.
      */
     fullWidth: React.PropTypes.bool,
 
     /**
-     * URL to link to when button clicked if `linkButton` is set to true.
+     * If `linkButton` is true, the URL to link to when the button
+     * is clicked.
      */
     href: React.PropTypes.string,
 
     /**
-     * Use this property to display an icon.
+     * The icon to be displayed within the button.
```

Comment:
Fair point. Since required props are called out explicitly, we probably don't need to mention that it's optional.

But 'An' rather than 'The' is less definitive I think.